sample/lib/ui/blocs/authentication_bloc/authentication_state.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter/material.dart';
+
+@immutable
+abstract class AuthenticationState extends Equatable {
+  AuthenticationState() : super();
+}
+
+class AuthenticationUninitialized extends AuthenticationState {
+  @override
+  String toString() {
+    return "AuthenticationUninitialized";
+  }
+
+  @override
+  List<Object> get props => [];
```

Comment:
Bunu parent class a alamiyomuyuz bos list doneceklerimizde? duplicate code dan uzaklasma falan filan :D

Reply:
bunu override etmek zorundayiz, her state'in kendi elemanlari farkli olabilir. abstract sinifa tasirsak beklenen gibi calismayabilir gibi geldi